data/feeds.yml

```diff
@@ -12,6 +12,11 @@
 #    tag or category to include gluster-related postings to keep
 #    things on topic :-)
 
+Gluster blog posts - @sacharya:
+  image: ant.png
+  rounded: false
+  feed: https://shwetha174.blogspot.com/2019/08/geo-repliaction-in-gluster.html
```

Comment:
This should be Atom/RSS feed URL, not the blog post URL

Reply:
`Control + U` or Right click and view source to get the feed URL

```
<link rel="alternate" type="application/atom+xml" title="Shwetha Acharya - Atom" href="https://shwetha174.blogspot.com/feeds/posts/default" />
```
Do not use this feed URL because it is common feed URL for all your blog posts. Create a catogary for Gluster in your blog website and add the feed url of that category